internal/provider/basic_auth_configuration_resource.go

```diff
@@ -52,8 +56,26 @@ func (r *basicAuthConfigurationResource) Schema(ctx context.Context, req resourc
 			"allow_users_to_signup_with_personal_email": schema.BoolAttribute{
 				Optional: true,
 				Description: "If true, your users will be able to sign up using personal email domains (@gmail.com, @yahoo.com, etc.)." +
-					"The default setting is true.",
+					"The default setting is true. This is only enabled if `signup_domain_allowlist` is empty.",
 			},
+			// "signup_domain_allowlist_enabled": schema.BoolAttribute{
+			// 	Optional:    true,
+			// 	Description: "If true, only users with email domains in the allowlist will be able to sign up. The default setting is false.",
+			// },
+			"signup_domain_allowlist": schema.ListAttribute{
+				ElementType: types.StringType,
+				Optional:    true,
+				Description: "A list of email domains that are allowed to sign up. This is only used if `signup_domain_allowlist_enabled` is true.",
```

Comment:
Can we remove any references in these field docstrings to the now deleted fields such as `signup_domain_allowlist_enabled`?